core/src/main/java/net/sourceforge/jnlp/runtime/Boot.java

```diff
@@ -346,6 +346,14 @@ private static String getMainFile() throws InvalidArgumentException {
         return null;
     }
 
+    private static String stripDoubleQuote(String path) {
+        if (path.length() >= 2 && IcedTeaWebConstants.DOUBLE_QUOTE.equals(path.charAt(0)) && IcedTeaWebConstants.DOUBLE_QUOTE.equals(path.charAt(path.length() - 1)));
```

Comment:
`IcedTeaWebConstants.DOUBLE_QUOTE` is a String
`path.charAt(0)` is a char

comparing String with char will always return false

Reply:
AutoBoxing/Unboxing doesn't work ?